<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="chartWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>801</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="autoStillLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AutoDistill Alpha v1.0</string>
    </property>
   </widget>
   <widget class="QWidget" name="logoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="autoMode">
     <attribute name="title">
      <string>AutoDistill Mode</string>
     </attribute>
     <widget class="QLabel" name="peakVaporTempLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Peak 째F:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="peakVaporTemp">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="currentVaporTemp">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="currentVaporTempLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Current 째F:</string>
      </property>
     </widget>
     <widget class="QLabel" name="BurnRateLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Burn Rate:</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="burnRateIndicator">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="autoStillHeadsRunSelect">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Heads Run</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="autoStillSpiritRunSelect">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Spirit  Run</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="manualMode">
     <attribute name="title">
      <string>Manual Burn Mode</string>
     </attribute>
     <widget class="QLabel" name="peakVaporTempLable2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Peak 째F:</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="burnRateIndicator2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="manualBurnRateUpButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="peakVaporTemp2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="currentVaporTemp2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="burnRateLable2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Burn Rate:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manualBurnRateDownButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentVaporTempLable2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Current 째F:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="calibrationMode">
     <attribute name="title">
      <string>Calibration Mode</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
